src/status_im/subs/wallet/wallet.cljs

```diff
@@ -392,6 +392,26 @@
           (assoc account :customization-color color))
         accounts)))
 
+(rf/reg-sub
+ :wallet/preferred-chains-for-address
+ :<- [:wallet/accounts]
+ :<- [:wallet/network-details]
+ :<- [:profile/test-networks-enabled?]
+ (fn [[accounts network-details test-networks-enabled?] [_ address]]
+   (let [preferred-chains-ids (some #(when (= (:address %) address)
+                                       (if test-networks-enabled?
+                                         (:test-preferred-chain-ids %)
+                                         (:prod-preferred-chain-ids %)))
```

Comment:
I almost forgot this is a set (which helps to perform the filter check below 😅 )